packages/database/src/core/RepoManager.ts

```diff
@@ -16,7 +16,7 @@
  */
 
 import { FirebaseApp } from '@firebase/app-types';
-import { FirebaseApp as FirebaseAppExp } from '@firebase/app-types-exp';
+import { FirebaseApp as FirebaseAppExp } from '@firebase/app-exp';
```

Comment:
I guess this should be added to database's deps (according to the failing check)